schemas/2018-05-01/Microsoft.Resources.json

```diff
@@ -4,21 +4,9 @@
   "title": "Microsoft.Resources",
   "description": "Microsoft Resources Resource Types",
   "resourceDefinitions": {
-    "resourceGroups": {
```

Comment:
>resourceGroups [](start = 5, length = 14)

we want to the "resourceGroups" type :)

Reply:
Thanks - looks like we need to fix this in swagger: https://github.com/Azure/azure-rest-api-specs/blob/master/specification/resources/resource-manager/Microsoft.Resources/stable/2018-05-01/resources.json

Reply:
This might be tricky.. the "resoruceGroups" API is in swagger, but it's not a "resource". We did special handling for it to deploy in template..

Reply:
Makes sense. In that case I think the best thing to do would be to pull it out in a separate file so that it's unaltered by the generation pipeline. I can make that change.